### PR TITLE
chore: windows bots via github actions

### PR DESCRIPTION
**Pull request by JoelEinbinder**
Mac will come in a follow up!

As a drive-by, removed `npm build` everywhere, because `npm install` already runs build.

### CODE REVIEW

**Review by aslushnikov — APPROVED** [review]
shots fired!

File: package.json [hunk 1]
```diff
@@ -13,18 +13,18 @@
...
+    "debug-test": "node --inspect-brk test/test.js",
     "test-doclint": "node utils/doclint/check_public_api/test/test.js && node utils/doclint/preprocessor/test.js",
     "test": "npm run lint --silent && npm run coverage && npm run test-doclint && node utils/testrunner/test/test.js",
     "prepare": "node prepare.js",
     "lint": "([ \"$CI\" = true ] && eslint --quiet -f codeframe --ext js,ts ./src || eslint --ext js,ts ./src) && npm run tsc && npm run doc",
     "doc": "node utils/doclint/cli.js",
-    "coverage": "cross-env COVERAGE=true npm run unit",
-    "fcoverage": "cross-env COVERAGE=true BROWSER=firefox npm run unit",
-    "wcoverage": "cross-env COVERAGE=true BROWSER=webkit npm run unit",
+    "ccoverage": "cross-env COVERAGE=true npm run ctest",
```
Comment (aesyondu) [inline package.json]: npm test is failing:

```
> playwright-core@0.11.1-post test /Users/aes/sandbox/playwright
> npm run lint --silent && npm run coverage && npm run test-doclint && node utils/testrunner/test/test.js

0 failures, 0 warnings.
DocLint Finished in 7.388 seconds
npm ERR! missing script: coverage
npm ERR! 
npm ERR! Did you mean one of these?
npm ERR!     ccoverage
npm ERR!     fcoverage
npm ERR!     wcoverage

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/aes/.npm/_logs/2020-02-18T23_57_11_890Z-debug.log
npm ERR! Test failed.  See above for more details.
```

Comment (aesyondu) [reply package.json]: Should test script be changed to:

```json
    "test": "npm run lint --silent && npm run ccoverage && npm run test-doclint && node utils/testrunner/test/test.js",
```

@JoelEinbinder

**Pull request merged**